contracts/FleekERC721.sol

```diff
@@ -278,6 +267,176 @@ contract FleekERC721 is ERC721, FleekAccessControl {
         emit NewTokenDescription(tokenId, _tokenDescription, msg.sender);
     }
 
+    /**
+     * @dev Add a new AccessPoint register for an app token.
+     * The AP name should be a DNS or ENS url and it should be unique.
+     * Anyone can add an AP but it should requires a payment.
+     *
+     * May emit a {NewAccessPoint} event.
+     *
+     * Requirements:
+     *
+     * - the tokenId must be minted and valid.
+     *
+     * IMPORTANT: The payment is not set yet
+     */
+    function addAccessPoint(uint256 tokenId, string memory apName, string memory source) public payable {
+        // require(msg.value == 0.1 ether, "You need to pay at least 0.1 ETH"); // TODO: define a minimum price
```

Comment:
Maybe we can make this a conditional.  If a global setting for payments is configured it uses that, otherwise it doesn't.

Reply:
Maybe we could address this to an issue and create other branch to work on it?

Reply:
Agreed